lib/scala/kg/src/main/scala/stores/kg/Neo4jKgStore.scala

```diff
@@ -87,6 +89,8 @@ final class Neo4jKgStore @Inject()(configuration: Neo4jStoreConfiguration) exten
       val recordMap = record.asMap().asScala.toMap.asInstanceOf[Map[String, Object]]
       KgEdge(
         datasource = recordMap("edge.datasource").asInstanceOf[String],
+        datasources = recordMap("edge.datasources").asInstanceOf[String].split(" ").toList,
```

Comment:
I'd use the Kgtk delimiter (|) in the neo4j record since you know the datasource is not going to contain that. In all likelihood a datasource won't have a space, either, but the spec doesn't guarantee that.

Reply:
Got it changed